django/test/runner.py

```diff
@@ -931,6 +938,7 @@ def build_suite(self, test_labels=None, extra_tests=None, **kwargs):
                     subsuites,
                     processes,
                     self.failfast,
+                    self.debug_mode,
```

Comment:
We should be able to add a regression test for this, see `NoInitializeSuiteTestRunnerTests` and `TestRunnerInitializerTests` in `tests/test_runner/tests.py`.

Reply:
Would you be able to look at this? I'm gonna be busy the rest of the day with client work. Next time I'll likely be able to work on it is tomorrow morning.

Reply:
I can have a look after lunch @adamchainz — thanks for your work! 🏅

Reply:
TBH I'm struggling to come up with a test any more sophisticated that the already existing `test_debug_mode_flag_overrides_settings`... 

```
        with self.settings(DEBUG=False):
            runner = DiscoverRunner(verbosity=0, interactive=False, debug_mode=True)
            self.assertTrue(runner.debug_mode)
```

The trouble is it's just a pass through param... 🤔

Reply:
I added a regression test.